app/src/androidTest/kotlin/at/bitfire/davdroid/resource/LocalCalendarTest.kt

```diff
@@ -67,7 +66,7 @@ class LocalCalendarTest {
 
     @After
     fun tearDown() {
-        calendar.delete()
```

Comment:
In theory, we could leave `delete()` here, because it's clear and legit what it does and there's no need to use the `LocalCollection` interface (the test is about the specific `LocalCalendar` class anyway).